book/src/contributing/using-dev-container.md

```diff
@@ -0,0 +1,55 @@
+# Development in Dev Containers
```

Comment:
FWIW I would match the title of the file with the name of the file

We also need to add this file to the book index